pkg/hub/client/local/client_test.go

```diff
@@ -0,0 +1,274 @@
+package local
+
+import (
+	"context"
+	"os"
+	"regexp"
+	"strings"
+	"testing"
+
+	"capact.io/capact/internal/cli/heredoc"
+	"capact.io/capact/internal/ptr"
+	gqllocalapi "capact.io/capact/pkg/hub/api/graphql/local"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// TODO(review): THIS FILE WILL BE REMOVED BEFORE MERGING. IT WAS ADDED ONLY FOR DEMO/PR TESTING PURPOSES.
```

Comment:
I know that is only for testing purposes of the PR and we don't want to keep it in the repository but on the other hand, seems valid and covers a lot of cases. It could be handy when someone would like to change something in the validation error and want to make sure that regression is fine.

Reply:
Fixed 👍